user_manual/android_app.rst

```diff
@@ -228,7 +228,7 @@ Uploading Files Taken From the Camera
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Pictures and videos can be uploaded from your smartphone after choosing the folder where they are stored.
-To specify where they are located, in the *"Settings"* options, under `Instant Uploads`_, enable one of *"Instant picture uploads"* or *"Instant video uploads"*. 
+To specify where they are located, in the *"Settings"* options, under `Camera uploads`_, enable one of *"Instant picture uploads"* or *"Instant video uploads"*. 
```

Comment:
We have to replace also this "Instant picture/video uploads" occurrences

Reply:
Thanks for catching that!